### PR TITLE
Add outliers beta documentation

### PR DESCRIPTION
**Pull request by ddaniels888**

### CODE REVIEW

**Comment by alq666** [comment]
I went back and forth on whether to include details on the algorithms we've chosen and I think we can provide details.

File: content/guides/outliers.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+<img src="/static/images/outliers/outliers-metric-alert.png" style="width:100%; border:1px solid #777777"/>
+
+## How to Use Outlier Detection on Your Data
+
+Starting in the beta period, we’ve added a new query function called `outliers` to our query language. This function acts as a filter, selecting only the series in a group that are outliers compared to the rest of the group.
+
+You can use this function to display and alert on outliers in your data. To try it out, you’ll first need a metric for which a group of hosts (or availability zones, partitions, etc) should exhibit uniform behavior. For the function to work, be sure that there are at least 3 or more members in the group. Given that, here are two ways to use outlier detection on that group.
```
Comment (githomin) [inline content/guides/outliers.md]: to display "an" alert

File: content/guides/outliers.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+
+### 1. Show Outliers in Dashboards
```
Comment (githomin) [inline content/guides/outliers.md]: ...in Dashboards (or Screenboards)

File: content/guides/outliers.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+{: #dashboards}
+
+For example, here is a graph of gunicorn requests by host with outlier detection enabled:
+
+<img src="/static/images/outliers/outliers-graph-dbscan-gunicorn.png" style="width:100%; border:1px solid #777777"/>
+
+You can see that one of the series is an outlier: it is handling significantly lower traffic than the others for the time window in question.
+
+During the beta period, to setup an outlier detection graph for your data you add two copies of the same metric to the graph: one showing all series in the group in a lightweight, greyscale color palette, and the other filtering to show outlier series only in a bold, warm color palette.
```
Comment (githomin) [inline content/guides/outliers.md]: to "set up" an outlier detection graph

File: content/guides/outliers.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+We use a simplified form of DBSCAN to detect outliers; any point that is not in the largest cluster will be considered an outlier. We consider each host to be a d-dimensional vector, where d is the number of values in the time series. Any hosts that are within a distance of `alpha x threshold` of each other will get clustered together.
+
+The threshold is calculated as follows. We create a new median time series by taking the median of the values from the existing time series at every time point. Then we calculate the distance between each host and the median series. The threshold is the median of those distances.
+
+In pseudocode:
+
+~~~ python
+    median_series = pointwise_median(series_list)
+    dists = [dist(series,median_series for series in series_list]
```
Comment (githomin) [inline content/guides/outliers.md]: dists = [dist(series,median_series"**)**" for series in series_list]

File: content/guides/outliers.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+
+<img src="/static/images/outliers/outliers-dbscan-banding.png" style="width:100%; border:1px solid #777777"/>
+
+Meanwhile a higher setting of alpha only identifies the host handling significantly less traffic as an outlier.
+
+<img src="/static/images/outliers/outliers-dbscan-no-banding.png" style="width:100%; border:1px solid #777777"/>
+
+### Median Absolute Deviation (MAD)
+
+The Median Absolute Deviation (MAD) algorithm is a robust means of calculating variability among data. It is designed to be usable for both normal and non-normal distributions.
```
Comment (githomin) [inline content/guides/outliers.md]: [Median Absolute Deviation (MAD)](https://en.wikipedia.org/wiki/Median_absolute_deviation)

Comment (githomin) [inline content/guides/outliers.md]: s/means/method/ (Could be confused with 'mean')

File: content/guides/outliers.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+
+We use the MAD to calculate an *outlier_factor* which we will compare to points from a series. If the percentage of points in the series that are above the *outlier_factor* is greater than some given threshold, then that series will be considered an outlier.
```
Comment (githomin) [inline content/guides/outliers.md]: s/the MAD/MAD/

File: content/guides/outliers.md (hunk 1 above)
Comment (ddaniels888) [inline content/guides/outliers.md]: I want "and" here to show that you can display (dashboards) and alert (monitors) on outliers

File: content/guides/outliers.md (hunk 2 above)
Comment (ddaniels888) [inline content/guides/outliers.md]: +1

File: content/guides/outliers.md (hunk 3 above)
Comment (ddaniels888) [inline content/guides/outliers.md]: +1

File: content/guides/outliers.md (hunk 4 above)
Comment (ddaniels888) [inline content/guides/outliers.md]: +1

File: content/guides/outliers.md (hunk 5 above)
Comment (ddaniels888) [inline content/guides/outliers.md]: +1

File: content/guides/outliers.md (hunk 6 above)
Comment (ddaniels888) [inline content/guides/outliers.md]: +1

File: content/guides/outliers.md (hunk 1 above)
Comment (githomin) [inline content/guides/outliers.md]: oh whoops. I misread that; sorry!

**Pull request merged**